packages/content/src/graphql/mutations/AddResourceToQueue.graphql

```diff
@@ -1,3 +0,0 @@
-mutation AddResourceToQueue($resourceType: ContentKind, $resourceId: ID!) {
```

Comment:
Not sure we'd want to remove this, as other packages such as `Catalog` and `FeaturedContent` may rely on it.

Reply:
change reverted